src/controls/SessionSheet.tsx

```diff
@@ -54,6 +54,20 @@ const SessionSheet: FC<SessionSheetProps> = ({
     editSession(reference, session);
   };
 
+  const shouldShowDescription = () => {
+    console.log(`Session ${session.name} has ${session.description.length} description length`);
+    return session.description.length > 0;
+  }
+
+  const shouldShowSpeakers = () => {
```

Comment:
This should be memoized instead of a function, calling this a ton of times is unnecessary and a performance hit.